CONTRIBUTING.md

```diff
@@ -74,6 +74,25 @@ If you are new to Git, these links might help:
 
 ---
 
+## Simple all-in-one localhost installation
+```
+git clone https://github.com/openshift/openshift-ansible
+cd openshift-ansible
+ansible-playbook -i inventory/byo/hosts.localhost playbooks/byo/config.yml
```

Comment:
Inventory should go in /inventory/hosts.localhost to go along with https://github.com/openshift/openshift-ansible/pull/6399
Should use playbooks/deploy_cluster.yml.